packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_fr_FR.json

```diff
@@ -60,6 +60,7 @@
   "privateDatabase_configurations_general_postgresql": "Configuration générale de PostgreSQL",
   "privateDatabase_configurations_general_postgresql_soon": "Bientôt disponible pour PostgreSQL",
   "privateDatabase_configuration_alert": "Toute modification entraîne un redémarrage du serveur.",
+  "privateDatabase_configuration_documentation": "Plus d'informations sur ces options : <a href=\"https://docs.ovh.com/fr/hosting/configurer-optimiser-son-serveur-de-base-de-donnees/#modifier-la-configuration-de-mon-serveur-de-bases-de-donnees\">https://docs.ovh.com/</a>",
```

Comment:
the link should be a parameter passed using angular-translate-values and should be different depending on user country

Reply:
Ideally also output the html from the translation file.